### PR TITLE
docs/transparent-validation/rte/v1.1.0/: add guides for using RTE

### PR DESCRIPTION
**Pull request by Pokisiekk**
Signed-off-by: Kamil Pokornicki <kamil.pokornicki@3mdeb.com>

### CODE REVIEW

File: docs/transparent-validation/rte/v1.1.0/quick-start-guide.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+* Micro-USB 5V 2.5A power supply
+* Orange Pi
+* SD card
+* Ethernet cable
+* USB-UART converter with 3 wire cables
+* Ubuntu (based on 22.04)
+
+## Preparation of RTE
+
+1. Build RTE image using the
```
Comment (sulewskiprzemyslaw) [inline docs/transparent-validation/rte/v1.1.0/quick-start-guide.md:20]: Do we really need to build an image from the repository? Can't we use the latest released version?

File: docs/transparent-validation/rte/v1.1.0/quick-start-guide.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    [meta-rte repository](https://github.com/3mdeb/meta-rte).
+1. Flash SD card using this image. The flashing method isn't imposed but an
```
Comment (sulewskiprzemyslaw) [inline docs/transparent-validation/rte/v1.1.0/quick-start-guide.md:22]: Why do we recommend `etcher`? In my opinion, the better option is to re-write this point similar to the point about flashing in the `SD-wire` documentation.

File: docs/transparent-validation/rte/v1.1.0/quick-start-guide.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    [etcher](https://www.balena.io/etcher/) is recommended.
+1. Insert SD card into Orange Pi.
+1. Insert Orange Pi into RTE.
```
Comment (sulewskiprzemyslaw) [inline docs/transparent-validation/rte/v1.1.0/quick-start-guide.md:25]: In my opinion, we should swap points regarding mounting the device. Firstly, we should write about connecting OrangePi to the RTE Hat, after that we might put the SD card into the slot on the OrangePi.

File: docs/transparent-validation/rte/v1.1.0/quick-start-guide.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    | RXD           | TX           |
+
+1. Open the serial connection with RTE from your PC using a previously connected
+    USB-UART converter by executing the following command:
+
+    ```bash
+    sudo minicom -D /dev/ttyUSB<x>
+    ```
+
+    Substitute `<x>` with the device number corresponding to your USB-UART
```
Comment (sulewskiprzemyslaw) [inline docs/transparent-validation/rte/v1.1.0/quick-start-guide.md:44]: In my opinion, the better option is to write about the `dmesg` command that allows us to identify the latest connected devices. Based on this info, the user should be able to determine the correct USB interface.

File: docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+* Two jumpers
+* Choose one of the following cables depending on the connectivity to your
+    platform:
+    - RS232-RJ45 cable
+    - RS232-RS232 cable
+    - 3 wire cables
+
+## Setting serial connection
+
+1. Connect the RTE header (J14 or J18) with the device serial port.
```
Comment (sulewskiprzemyslaw) [inline docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md:18]: In my opinion, the first point in this section should describe the determination method for connector type:
1. DUT has a serial port -> needs to use an RS232-RS232 cable.
2. DUT has the debug port in the form of the RJ45 -> needs to use an RS-232-RJ45 cable.
3. DUT does not have port, but have pins for serial connection on the board -> need to use a 3-wire cable.

File: docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+1. Put the two jumpers on
```
Comment (sulewskiprzemyslaw) [inline docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md:19]: The putting jumpers section should correspond with the comment above about the determination of the method of connection.

Also, this point should be moved before the point on the connection to the RTE header.

File: docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    [RTE J16 Header](../specification/#uart-output-select-header):
+
+    1. For the RS232 port, they join COM and RS232 rows both on RX and TX
+        columns.
+    1. For the UART port, they join COM and EXT rows both on RX and TX
+        columns.
+
+1. Connect with RTE via `ssh` or `minicom`.
```
Comment (sulewskiprzemyslaw) [inline docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md:27]: There is a lack of information on what situation can be used 'ssh' and in which situation 'minicom' is preferred for connection.

File: docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+1. There are two ways to open a serial connection:
```
Comment (sulewskiprzemyslaw) [inline docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md:28]: If there's an option to use the telnet, we should describe how to configure `ser2net` and on which depends it configuration.

File: docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+## Prerequisites
+
+* [Prepared RTE](../v1.1.0/quick-start-guide.md)
+* Two jumpers
+* Choose one of the following cables depending on the connectivity to your
+    platform:
+    - RS232-RJ45 cable
+    - RS232-RS232 cable
+    - 3 wire cables
```
Comment (sulewskiprzemyslaw) [inline docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md:14]: In some cases there's an option to use also USB-RS232 converter and 3-wire cable

File: docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+        ```bash
+        telnet 192.168.X.X 13541
+        ```
+
+        > To configure telnet, update the file `/etc/ser2net.conf`
+
+    1. Minicom - by executing the following command:
+
+        ```bash
+        minicom -D /dev/ttyS1 -o -b 115200
```
Comment (sulewskiprzemyslaw) [inline docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md:40]: Should be detailed in what situation the port (current: `ttyS1`) should be changed in this command.

File: docs/transparent-validation/rte/v1.1.0/flashing-guide.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+
+This document describes how to set up external flashing on some devices using
+RTE.
+
+## NS5x/7x 12th Generation
+
+### Prerequisites
+
+* [Prepared RTE](../v1.1.0/quick-start-guide.md)
+* WSON8 probe - can be bought from China on Aliexpress or eBay.
```
Comment (sulewskiprzemyslaw) [inline docs/transparent-validation/rte/v1.1.0/flashing-guide.md:11]: Does the information about, where the probe might be bought necessary?

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/transparent-validation/rte/v1.1.0/quick-start-guide.md (hunk 1 above)
Comment (macpijan) [reply docs/transparent-validation/rte/v1.1.0/quick-start-guide.md:20]: Good point if we insutrct someoe to build the image RTE OS, no one will use that documentation essentialy.

Building RTE OS is complex and time consuming task.

Comment (Pokisiekk) [reply docs/transparent-validation/rte/v1.1.0/quick-start-guide.md]: No, I didn't think images were so readily available, I corrected it

File: docs/transparent-validation/rte/v1.1.0/quick-start-guide.md (hunk 3 above)
Comment (Pokisiekk) [reply docs/transparent-validation/rte/v1.1.0/quick-start-guide.md:45]: Swaped

File: docs/transparent-validation/rte/v1.1.0/quick-start-guide.md (hunk 4 above)
Comment (Pokisiekk) [reply docs/transparent-validation/rte/v1.1.0/quick-start-guide.md]: Added info about `dmesg` command

File: docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md (hunk 5 above)
Comment (Pokisiekk) [reply docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md]: added

File: docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md (hunk 6 above)
Comment (Pokisiekk) [reply docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md:20]: Updated section

File: docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md (hunk 7 above)
Comment (Pokisiekk) [reply docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md]: Added some more info

File: docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md (hunk 8 above)
Comment (Pokisiekk) [reply docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md:46]: Added some more info

File: docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md (hunk 10 above)
Comment (Pokisiekk) [reply docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md:59]: Added

File: docs/transparent-validation/rte/v1.1.0/flashing-guide.md (hunk 11 above)
Comment (Pokisiekk) [reply docs/transparent-validation/rte/v1.1.0/flashing-guide.md]: No I guess, I removed it, I copied it from another document: https://docs.dasharo.com/variants/msi_z690/recovery/

**Pull request merged**